pkg/assets/isto-workspace-deploy.go

```diff
@@ -129,7 +129,7 @@ func deployService_accountYaml() (*asset, error) {
 	return a, nil
```

Comment:
This should be left out of the commit as well, no?

Reply:
434177f

Reply:
434177f